Enabling Third-party Smart Contract Account Applications on Polkadot Based on Account Abstraction.md

```diff
@@ -0,0 +1,159 @@
+# Enabling Interoperable Third-party Smart Contract Account Applications on Polkadot Based on Account Abstraction (AA) for Demand Abstraction
+
+- **Team Name:** Smarter AA Wallet
+- **Payment Address:** bc1quf6qljy5z07sxlv6fgv04zqzaq0g30z3v47v3u (BTC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+Please provide the following:
+
+The capacity, usability, and personalization of blockchain are limited by the existing crypto account system like externally-owned accounts (EOA) and contract accounts (CA). To address this issue, the recent significant account upgrade called account abstraction (AA) enables users to freely self-define the functions, logic, and settings of their crypto accounts via smart contracts. Compared with the existing EOA and CA, with AA users can have more account functionalities, higher customization, and better usage experience as good as Web2 while maintaining the decentralization of Web3. 
```

Comment:
What is this recent significant account upgrade? This sounds like an Ethereum thing.